lib/foreman_openscap/engine.rb

```diff
@@ -90,6 +90,9 @@ class Engine < ::Rails::Engine
                      :resource_type => 'ForemanOpenscap::ScapContent'
           permission :edit_hosts, { :hosts => [:openscap_proxy_changed] }, :resource_type => "Host"
           permission :edit_hostgroups, { :hostgroups => [:openscap_proxy_changed] }, :resource_type => "Hostgroup"
+          permission :view_tailoring_files, { :tailoring_files => [:index] }, :resource_type => 'ForemanOpenscap::TailoringFile'
```

Comment:
should be probably added to roles "Compliance viewer" and "Compliance manager" (others to manager role)